cra-server/index.js

```diff
@@ -21,8 +21,8 @@ const env = process.env.NODE_ENV || 'development';
 
 
 // ssl self signed certificate and key for browser session
-const cert = fs.readFileSync(path.join(__dirname, '../') + "ssl/keys/server.cert");
-const key = fs.readFileSync(path.join(__dirname, '../') + "ssl/keys/server.key");
+const cert = fs.readFileSync(path.join(__dirname, '../') + "ssl/keys/EgeriaReactUIClient.cert.pem");
```

Comment:
The server/client designation can be complex

My understanding is that cra-client is the code that runs in the client's browser?
Then cra-server runs in the express server (it is then an egeria chassis client)

So wouldn't this be EgeriaReactUIServer.cert.pem

Logically we've generated a certificate for each component to identify itself.

Note that in practice many SSL connections are setup one way -- so often the client side of a client/server connection will not present a certificate, it will only validate that the server component sends back. (And here client-server refers to that specific connection ie client initiating the connection. A client itself may be a server for the next hop). However using both adds the best security, and should be configurable.

Reply:
and same below with the key